cli/replicaset/cconfig.go

```diff
@@ -62,6 +63,14 @@ type CConfigInstance struct {
 	evaler connector.Evaler
 }
 
+// patchRoleTarget describes a content to patch a config.
+type patchRoleTarget struct {
+	// path is a destination to a patch target in config.
+	path []string
+	// roleNames are roles to add by path in config.
+	roleNames []string
```

Comment:
In what cases do we add multiple roles?

Reply:
It refers with update logic in yaml config. Since we have list type in roles value, we get parse it into a slice, add role if it is not exist and set it back by the path.

Reply:
```roleNames are roles to add by path in config``` > ```roleNames are roles to set by path in config``` than?

Reply:
Edit comment.